packages/runner/src/schema.ts

```diff
@@ -428,16 +432,19 @@ export function validateAndTransform(
 
   // TODO(seefeld): The behavior when one of the options is very permissive (e.g. no type
   // or an object that allows any props) is not well defined.
-  if (Array.isArray(resolvedSchema.anyOf)) {
-    const options = resolvedSchema.anyOf
-      .map((option) => {
-        const resolved = resolveSchema(option, rootSchema);
-        // Copy `asCell` over, necessary for $ref case.
-        if (option.asCell) return { ...resolved, asCell: true };
-        if (option.asStream) return { ...resolved, asStream: true };
-        else return resolved;
-      })
-      .filter((option) => option !== undefined);
+  if (
+    Array.isArray(resolvedSchema.anyOf) || Array.isArray(resolvedSchema.oneOf)
+  ) {
+    const options =
+      ((resolvedSchema.anyOf ?? resolvedSchema.oneOf)! as JSONSchema[])
+        .map((option) => {
+          const resolved = resolveSchema(option, rootSchema);
+          // Copy `asCell` over, necessary for $ref case.
+          if (option.asCell) return { ...resolved, asCell: true };
+          if (option.asStream) return { ...resolved, asStream: true };
+          else return resolved;
+        })
+        .filter((option) => option !== undefined);
```

Comment:
We shouldn't need this anymore once the updates to make CTS generate `anyOf` land